### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tbacea**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,15 +3,25 @@
...
 
 ## How to Run
+<ol>
+    <li>
+        <p>
+            Before you begin, make sure you have <a href="https://nodejs.org/en/download/current">Node.js</a> installed on your system.
+        </p>
+    </li>
+    <li>
+        <p>After that, go into the travel-assistant folder</p>
```
Comment (Mexaby) [inline README.md:13]: This is not necessary

File: README.md [hunk 2]
```diff
@@ -3,15 +3,25 @@
...
+        <code>cd .\travel-assistant\</code>
+    </li>
+    <li>
+        <p>and then you can set up the project with the following command:</p>
```
Comment (Mexaby) [inline README.md:17]: Include the directory change in this line

File: README.md [hunk 3]
```diff
@@ -3,15 +3,25 @@
...
+        <code>npm install</code>
+    </li>
+    <li>
+        <p>You may then run the project with the following command:</p>
```
Comment (Mexaby) [inline README.md:21]: Remove conditional and use affirmative statements e.g. run the project with:

File: README.md (hunk 1 above)
Comment (tbacea) [reply README.md:13]: it's necessary, as the project folder is not the root of the git repo

File: README.md (hunk 2 above)
Comment (tbacea) [reply README.md:17]: no

File: README.md (hunk 3 above)
Comment (tbacea) [reply README.md:21]: will do

File: README.md (hunk 2 above)
Comment (Mexaby) [reply README.md:17]: yes

Comment (tbacea) [reply README.md:17]: these are 2 different steps. you either modify the repo as with the issue mentioned above, or let the readme like this.

**Pull request closed without merging**